kernel/aster-nix/src/fs/procfs/filesystems.rs

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use alloc::format;
+
+use crate::{
+    fs::{
+        procfs::template::{FileOps, ProcFileBuilder},
+        utils::Inode,
+    },
+    prelude::*,
+};
+
+struct FileSystemType {
+    name: String,
+    nodev: bool,
```

Comment:
`nodev` is not a good name for a bool value. I suggest to use `has_nodev`

Reply:
Thanks. I feel like `is_nodev` is better. Any thoughts?